_security-analytics/usage/findings.md

```diff
@@ -37,12 +37,31 @@ As one more alternative, you can select an option from the **Recently used date
-* The bell icon allows you to open the Create detector alert trigger pane, where you can quickly set up an alert for the specific finding and modify rules and their conditions as required.
+
+The **Actions** column includes two options for each finding:
+* The diagonal arrow provides a way to open the **Finding details** pane, which describes the finding by parameters defined when creating the detector and includes the document that generated the finding.
+* The bell icon allows you to open the **Create detector alert trigger** pane, where you can quickly set up an alert for the specific finding and modify rules and their conditions as required.
 For details on setting up an alert, see [Set up alerts]({{site.url}}{{site.baseurl}}/security-analytics/sec-analytics-config/detectors-config/#step-3-set-up-alerts) in detector creation documentation.
 
+Each finding in the list also includes a **Finding ID**. You can select the ID as another option for opening the **Finding details** pane. An example of **Finding details** is shown in the following image.
+
+<img src="{{site.url}}{{site.baseurl}}/images/Security/findings1.png" alt="Finding details pane" width="60%">
+
+### Viewing surrounding documents
+
+The **Finding details** pane contains specific information about the finding, including the document that generated the finding. To investigate the series of events that led to the finding or followed the finding, you can select **View surrounding documents** to open the document in the **Discover** panel and view other documents preceding or following it.
+
+1. Open **Finding details** by selecting the **Finding ID** in the list of findings.
+1. In the **Documents** section, select **View surrounding documents**. If an index pattern already exists for the document, the **Discover** panel opens and displays the document. If an index pattern does not exist, the **Create index pattern to view documents** window opens and prompts you to create an index pattern, as shown in the following image.
+<br><img src="{{site.url}}{{site.baseurl}}/images/Security/findings2.png" alt="popup window prompting users to create an index pattern" width="60%">
+1. In the **Create index pattern to view documents** window, the index pattern name is automatically populated. Enter the appropriate time field from the log index used to determine the timing for log events. For information on mapping log fields to detector fields, see [Step 2. Create field mappings]({{site.url}}{{site.baseurl}}/security-analytics/sec-analytics-config/detectors-config/#step-2-create-field-mappings). Select **Create index pattern**. The **Create index pattern to view documents** confirmation window opens.
+1. Select **View surrounding documents** in the confirmation window. The **Discover** panel opens, as shown in the following image.
+<br><img src="{{site.url}}{{site.baseurl}}/images/Security/findings4.png" alt="popup window prompting users to create an index pattern" width="85%">
```

Comment:
Remove `<br>` and add a new line before and after the image.

Reply:
Also, change alternate text in the image.

Reply:
RE: <br> as above.

Good catch. Changed alternate text.